util/regexp-assemble/tests/fixtures.py

```diff
@@ -1,24 +1,26 @@
-import os, uuid
-from pathlib import Path
+import uuid
+from pathlib import Path, PurePath
 import pytest
 
 from lib.context import Context
 
 @pytest.fixture
 def include_file(context, include_file_name):
-    file_path = os.path.join(context.include_files_directory, include_file_name)
+    file_path = PurePath.joinpath(context.include_files_directory, include_file_name)
```

Comment:
Use `Path` here directly, instead of creating a `Path` on the next line.

Reply:
Done.